template/project.yml

```diff
@@ -15,21 +15,30 @@ project:
   # These are random strings used to configure the hash salt.
   hash_salt: ${project.hash_salt}
 
-# Configuration settings for new git repository.
-git:
-  default_branch: master
-  remotes:
-    # Defining git remotes allows builds deployed via CI.
-    - bolt8@svn-5223.devcloud.hosting.acquia.com:bolt8.git
-
```

Comment:
This test, `tests/phpunit/DeployTest.php` line 61 is looking for the variables you are deleting here and fails,

```
foreach ($this->config['git']['remotes'] as $remote) {
```